spec/features/user_can_checkout_spec.rb

```diff
@@ -29,5 +29,27 @@
       expect(page).to have_content("70-200 Canon L")
       expect(page).to have_content("portrait lens")
     end
+
+    it "can checkout" do
+      login_user
+      add_item_to_cart
+      visit cart_path
+      fill_in "days", with: 5
+      click_button "Pay with Card"
+
+      sleep(2)
+      stripe_iframe = all('iframe[name=stripe_checkout_app]').last
+      Capybara.within_frame stripe_iframe do
+        page.execute_script(%Q{ $('input#email').val('blah@blah.com'); })
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.